src/wallet-instance-attestation/issuing.ts

```diff
@@ -38,6 +38,40 @@ async function getAttestationRequest(
     .sign();
 }
 
+/**
+ * Validate a Wallet Instance Attestation token.
+ * Either return true or throw an exception.
+ *
+ * @param wia Signed Wallet Instance Attestation token
+ * @param walletProviderEntityConfiguration Entity Configuration object for the issuing Wallet Provider
+ * @returns The token is valid
+ * @throws {WalletInstanceAttestationIssuingError} When the received token fails to validate. This can happen due to invalid signature, espired token or malformed JWT token.
```

Comment:
```suggestion
 * @throws {WalletInstanceAttestationIssuingError} When the received token fails to validate. This can happen due to invalid signature, expired token or malformed JWT token.
```